intro.md

```diff
@@ -15,7 +15,7 @@ All of these solutions work together to solve a bigger problem.
 ## Design patterns in Rust
 
 There are many problems that share the same form.
-Due to the fact that Rust is not object-oriented design patterns vary with
+Due to the fact that Rust is not object-oriented, design patterns vary with
```

Comment:
```suggestion
Due to the fact that of Rust not being object-oriented, design patterns vary with
```

Better like this?

Reply:
Adding `of` doesn't make sense grammatically and `being` doesn't help here although it's not *incorrect*. 

Really, the phrase `Due to the fact that` is wordy. The sentence can be reworded to read:
> Since Rust is not an object-oriented language, design patterns vary with [...]

Reply:
Ok :)